website/docs/r/secretsmanager_secret_policy.html.markdown

```diff
@@ -27,10 +27,10 @@ resource "aws_secretsmanager_secret_policy" "example" {
   "Version": "2012-10-17",
   "Statement": [
 	{
-	  "Sid": "EnableAllPermissions",
+	  "Sid": "EnableAnotherAWSAccountToReadTheSecret",
 	  "Effect": "Allow",
 	  "Principal": {
-		"AWS": "*"
+		"AWS": "arn:aws:iam::111122223333:root"
```

Comment:
Thanks for the contribution @christophetd! I think this makes sense. Can we change the random account id to: `123456789012` as that's what we typically use.